locale/en_GB/translations.json

```diff
@@ -1343,6 +1343,8 @@
   "Toggle first emoji reaction on selected message": "Toggle first emoji reaction on selected message",
   "Toggle preview mode": "Toggle preview mode",
   "Toggle subscription": "Toggle subscription",
+  "Subscribe to this channel": "Subscribe to this channel",
+  "Unsubscribe from this channel": "Unsubscribe from this channel",
```

Comment:
Translations are handled via Transifex. Please don't update the `translations.json` manually.

Reply:
For additional reference see: https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations.